src/ray/raylet/worker_pool.cc

```diff
@@ -804,10 +804,7 @@ Status WorkerPool::RegisterDriver(const std::shared_ptr<WorkerInterface> &driver
     if (num_initial_python_workers_for_first_job_ > 0) {
       delay_callback = true;
       // Start initial Python workers for the first job.
-      for (int i = 0; i < num_initial_python_workers_for_first_job_; i++) {
-        PopWorkerStatus status;
-        StartWorkerProcess(Language::PYTHON, rpc::WorkerType::WORKER, job_id, &status);
-      }
+      PrestartWorkers(Language::PYTHON, num_initial_python_workers_for_first_job_);
```

Comment:
Currently the pre-start is triggered when the first job is registered. With your change, we can prestart as soon as raylet starts?

Reply:
yeah, we actually does it as part of the `ResizeIdleWorkers`, which runs periodically